src/main/java/org/zalando/nakadi/util/Aruha771MigrationHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.zalando.nakadi.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zalando.nakadi.config.JsonConfig;
+import org.zalando.nakadi.domain.Subscription;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * Temporary class to be used for DB migration of ARUHA-771
+ */
+public class Aruha771MigrationHelper {
+
+    private final HashGenerator hashGenerator;
+    private final ObjectMapper jsonMapper;
+    private final String url;
+    private final String user;
+    private final String pass;
+
+    public static void main(final String[] args) throws Exception {
+        final Aruha771MigrationHelper helper = new Aruha771MigrationHelper(
+                new HashGenerator(),
+                new JsonConfig().jacksonObjectMapper());
+        helper.fillSubscriptionsHashes();
+    }
+
+    public Aruha771MigrationHelper(final HashGenerator hashGenerator, final ObjectMapper jsonMapper) throws Exception {
+        this.hashGenerator = hashGenerator;
+        this.jsonMapper = jsonMapper;
+
+        url = "jdbc:postgresql://localhost:5432/local_nakadi_db";
+        user = "nakadi";
+        pass = "nakadi";
+        Class.forName("org.postgresql.Driver").newInstance();
+    }
+
+    public void fillSubscriptionsHashes() throws Exception {
+
+        try (Connection con = DriverManager.getConnection(url, user, pass)) {
+            final Statement st = con.createStatement();
+            final ResultSet rs = st.executeQuery("SELECT s_subscription_object FROM zn_data.subscription");
+
+            int i = 0;
+            while (rs.next()) {
+                final String jsonSubscription = rs.getString("s_subscription_object");
+                final Subscription subscription = jsonMapper.readValue(jsonSubscription, Subscription.class);
+                final String hash = hashGenerator.generateSubscriptionKeyFieldsHash(subscription);
+
+                final Statement statement = con.createStatement();
+                statement.execute("UPDATE zn_data.subscription SET s_key_fields_hash='" + hash + "' " +
+                        "WHERE s_id='" + subscription.getId() + "'");
+
+                System.out.println(++i + " | " + subscription.getId() + " | " + hash);
```

Comment:
1) It's difficult to do as it requires:
a) Nakadi JsonConfig
b) HashGenerator
c) Subscription domain class
2) For me this class is just a script like sql scripts in https://github.com/zalando/nakadi/tree/master/database/migration - but it is written in different language than SQL.

I agree that I maybe put it to wrong place in our repo so that people can think that it is part of Nakadi source code. I will put it to another source set.